gpustack/policies/candidate_selectors/sglang_resource_fit_selector.py

```diff
@@ -183,6 +177,11 @@ def _validate_arguments(self):
         if speculative_algorithm is not None and enable_mixed_chunk:
             raise ValueError("enable_mixed_chunk is required for speculative decoding")
 
+        try:
+            self._check_tp_size_divisibility(tp_size)
```

Comment:
Why do we need to call _check_tp_size_divisibility function twice in _validate_arguments?